<launch>

<arg name="sonarPort" value="5000" />
<arg name="sonarAddress" value="192.168.0.88"/>


<!-- <arg name="loggerPath"/> //-->

<!-- <node pkg="logger_binary" name="logger_binary" type="logger_binary" output="screen" args="$(arg loggerPath)"/> //-->

<!-- <node pkg="logger_text" name="logger_text" type="logger_text" output="screen" args="$(arg loggerPath)"/> //-->

<!-- <node pkg="gnss_dummy" name="gnss_dummy" type="gnss_dummy" output="screen"/> //-->

<node pkg="sonar_nmea_0183_tcp_client" name="sonar" type="sonar_nmea_0183_tcp_client" output="screen" args="$(arg sonarAddress) $(arg sonarPort)"/>

<node pkg="imu_dummy" name="imu_dummy" type="imu_dummy" output="screen"/>

<node pkg="hydroball_control_websocket" name="hydroball_control_websocket" type="hydroball_control_websocket"/>


</launch>
